test/index.test.js

```diff
@@ -325,10 +325,11 @@ test('test catch of invalid image data error', (done) => {
     .use(sharp())
     .build((err, files) => {
       if (err) {
+        console.log(err);
```

Comment:
No need to extra log the error

Reply:
No it isn't needed, forgot to remove it after testing.